WoWDatabaseEditor.Common/WDE.CMMySqlDatabase/Models/ConditionLine.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using LinqToDB.Mapping;
+using WDE.Common.Database;
+
+namespace WDE.CMMySqlDatabase.Models
+{
+    [ExcludeFromCodeCoverage]
+
+    /// <summary>
+    /// Condition System
+    /// </summary>
+    [Table("conditions")]
+    public class ConditionLineWoTLK : IConditionLine
+    {
```

Comment:
As I've mentioned before, I think the condition systems are so different, that it will be easier to make another interface for cmangos conditions. (but there is no need to do it now, just skip this model for now I think).

Reply:
removed 👍